### ISSUE TITLE
Testing for November 2018 release

### ISSUE THREAD

**Issue by garemoko** [davidpesce/moodle-mod_tincanlaunch#160]
@davidpesce @nadavkav @mediabounds @thepurpleblob @caperneoignis

I'm planning to make a new release of this plugin soon and have pushed a number of changes to the master branch. I would ideally like to have the following tested before release:

- [x] A clean install
- [x] Upgrade from the [latest version on plugins directory](https://moodle.org/plugins/download.php/12928/mod_tincanlaunch_moodle34_2016122702.zip)
- [ ] Upgrade from an [earlier version before 2016121200](https://moodle.org/plugins/download.php/9049/mod_tincanlaunch_moodle29_2015033100.zip)
- [x] Watershed integration with paid tier account
- [x] Watershed integration with Essentials account

I plan to test these myself, except for the upgrade tests. Additional testing is appreciated. 

Completion tracking is out of scope for this release. I know that there are significant issues with it (#125 #126 #134) and for now at least enabling completion tracking is not supported.

**Comment by garemoko** [comment]
Seems to all work for me. 

@davidpesce @nadavkav @mediabounds @thepurpleblob @caperneoignis Please can you let me know in the next 48 hours if you plan to help out with testing this. If you do, I can wait but if not I'll go ahead and cut a release.

**Comment by garemoko** [comment]
I pushed a release candidate with the intention of pushing a stable release after testing. https://moodle.org/plugins/pluginversion.php?id=18130

**Comment by davidpesce** [comment]
I won’t be able to until Friday at the earliest. Still at moot and flying back home tomorrow.

**Comment by mediabounds** [comment]
Sorry, @garemoko — I’m not actively working in Moodle right now so I won’t be able to help test this.

**Comment by garemoko** [comment]
Note: we made some changes to the Watershed session API that will break the release candidate. The release candidate should work with watershed production right now. The Github master branch should work with Watershed sandbox. I'm anticipating a couple more changes on the Watershed side and will do a new release of this plugin once that's all finalized on watershed production.

**Comment by davidpesce** [comment]
Any chance you can remove the LL integration? Since it's only working with basic for the time being.

**Comment by garemoko** [comment]
@davidpesce doesn't it still work with Learning Locker 1?

**Comment by garemoko** [comment]
The plugin should now be stable in regards to Watershed integration changes (though still waiting on updates to Watershed itself to be pushed from sandbox to production). 

I'll release a new version once we answer the Learning Locker question. I'm fine with removing it if it's no longer useful.

**Comment by davidpesce** [comment]
Oh, yea, it definitely works on LLv1. I still have a few clients using a customized version of it. Maybe it should specify that it's LLv1?

**Comment by davidpesce** [comment]
I'll check with @ryansmith94 and see about what is needed to get LLv2 working in here.

**Comment by caperneoignis** [comment]
I can test it. but I have made changes to the plugin, which I have included in the issues. So I'd have to pull a new instance to test, and only for testing locally but I can make it happen probably next week the earliest though. @garemoko sorry just seeing this now, which github did push notifications to my phone.

**Comment by garemoko** [comment]
@davidpesce I updated the language file so it should say Learning Locker 1 now. 

As for what's needed for LL2, see https://github.com/garemoko/moodle-mod_tincanlaunch/issues/144#issuecomment-344413706 for some pretty clear instructions. 

@caperneoignis thanks. I'm going to go ahead with a release of the plugin as we have in master now, but if you notice issues (aside from with completions) I may be able to incorporate them into another release next week.

**Comment by garemoko** [comment]
Calling this done, but please continue testing.
